ddl/ddl_api.go

```diff
@@ -423,12 +375,28 @@ func setCharsetCollationFlenDecimal(tp *types.FieldType, specifiedCollates []str
 }
 
 // outPriKeyConstraint is the primary key constraint out of column definition. such as: create table t1 (id int , age int, primary key(id));
```

Comment:
Please update this comment. The format needs to be like `buildColumnAndConstraint  ....`.

Reply:
OK